README.md

```diff
@@ -107,6 +107,57 @@ DaskTaskRunner(
 )
 ```
 
+### Distributing Dask collections across workers
+
+If your task contains a Dask collection, such as a `dask.DataFrame` or `dask.Bag`, to distribute the work across workers and achieve parallel computations, use the `get_dask_client` context manager.
+
+Within task run contexts:
+```python
+import dask
+from prefect import flow, task
+from prefect_dask import DaskTaskRunner, get_dask_client
+
+@task
+def compute_task():
+    with get_dask_client(timeout="120s") as client:
```

Comment:
@ahuang11 is there a reason you've set `timeout` in the examples? By showing it this way, it seems like the user always needs to set the timeout value (or that 2 minutes is a good one), when Dask already has a default value set in `distributed.comm.timeouts.connect`.

I'd suggest noting `get_dask_client` accepts the same args as `worker_client` and an admonition/call-out showing people how to set `timeout` for the flow example below, since that might be a bit trickier to figure out.

Reply:
I refactored the internals. `timeout` should work the same in task and flow runs now!